models/examples/example-2-acute-nominated-pharmacy/PrepareRequest-FhirMessageUnsigned.json

```diff
@@ -77,7 +77,7 @@
         "courseOfTherapyType": {
           "coding": [
             {
-              "system": "https://fhir.nhs.uk/R4/CodeSystem/UKCore-PrescriptionType",
+              "system": "https://fhir.nhs.uk/R4/CodeSystem/medicationrequest-course-of-therapy",
```

Comment:
I think the system is http://terminology.hl7.org/CodeSystem/medicationrequest-course-of-therapy for this case